connect/src/routes/route.ts

```diff
@@ -130,6 +130,7 @@ export abstract class ManualRoute<
   NATIVE_GAS_DROPOFF_SUPPORTED = false;
   IS_AUTOMATIC = false;
   public abstract complete(sender: Signer, receipt: R): Promise<R>;
+  public abstract resume(tx: TransactionId): Promise<R>;
```

Comment:
Should this go on the base class `Route` since we may want to `resume` a transaction for any route? For example, an auto NTT transfer may need to be manually resumed to complete the transfer if it was queued on the destination chain.

Reply:
will make resume-able a new abstract class

Reply:
(decided to keep this as-is)